types/graphql/language/parser.d.ts

```diff
@@ -11,7 +11,10 @@ export interface ParseOptions {
    * in the source that they correspond to. This configuration flag
    * disables that behavior for performance or testing.
    */
-  noLocation?: boolean;
+  noLocation?: boolean,
+  allowLegacySDLEmptyFields?: boolean,
+  allowLegacySDLImplementsInterfaces?: boolean,
+  experimentalFragmentVariables?: boolean,
```

Comment:
@colepatrickturner Can you please also copy options descriptions:
https://github.com/graphql/graphql-js/blob/98751e473fd7a02c514af48218d87d26daadb5f5/src/language/parser.js#L79-L115